index.html

```diff
@@ -1,14 +1,322 @@
+            maiores, ullam omnis impedit libero eveniet deleniti atque error?</p>
+        </div>
+        <hr>
+        <div class='something-post'>
+
+          <div>
+            <h3 class='something-title'>SOMETHING OF NOTE</h3>
+            <div id='text-wrap'>
+              <img src='Photos\pexels-photo-110874.jpeg' title="Stock Image Two" alt="Stock Image Two from Pexels"
+                style="height:100px; width:100px;"> </div>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum at enim, debitis quibusdam
+              distinctio
+              voluptatibus harum blanditiis temporibus optio, officiis mollitia placeat recusandae dignissimos
+              non, eos
+              quisquam sit libero sequi!</p>
+            <a href='./learn-more.html'><button class='tell-me-button'>Learn More</button></a>
+          </div>
+          <div id='line'>
+          </div>
+          <div>
+            <h3 class='something-title'>SOMETHING OF LESS NOTE</h3>
+            <div id='text-wrap'>
+              <img src='Photos\pexels-photo-110874.jpeg' title="Stock Image Two" alt="Stock Image Two from Pexels"
+                style="height:100px; width:100px;"> </div>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum at enim, debitis quibusdam
+              distinctio
+              voluptatibus harum blanditiis temporibus optio, officiis mollitia placeat recusandae dignissimos
+              non, eos
+              quisquam sit libero sequi!</p>
+
+            <a href='./learn-more.html'><button class='tell-me-button'>Learn More</button></a>
+          </div>
+        </div>
+      </section>
+  </div>
+
+
+
+
+  <!-- Footer Section -->
+  <section id='entire-footer'>
+    <div id='footer-title'>
+      <h1>
+        QUESTIONS OR COMMENTS?<span id='blue-color' class='big-footer-font'> GET IN TOUCH</span>:
+      </h1>
+    </div>
+    <div id='content-wrapper'>
+      <div id='left'>
+        <section id="footer">
+
+          <div id='footer-content'>
+            <div id='footer-contact-info'>
+              <form id='footer-form-container'>
+                <textarea id="name-email" name="name-email" rows="1" cols="20">
+            NAME
+            </textarea>
+              </form>
+              <form id='footer-form-container-2'>
+                <textarea id="name-email" name="name-email" rows="1" cols="19">
+              EMAIL
+              </textarea>
+              </form>
+            </div>
+          </div>
+        </section>
+        <section id="footer">
+
+          <div id='footer-content'>
+            <div id='footer-contact-info'>
+              <form id='footer-form-container'>
+                <textarea id="Message" name="Message" rows="10" cols="43">
+            MESSAGE
+            </textarea>
+              </form>
+            </div>
+            <button class='tell-me-button'>Send Message</button>
+          </div>
+        </section>
+      </div>
+
+

```

Comment:
Ah the footer.  Everyone's favorite.  Great basic layout.  Your flex-containers are there: the orientation is spot on! Your main issue is the same thing that happened with your `sidebar`.  Nothing serious, just mostly an issue with the width of your text container.  Which is caused by a series of great choices that conflict each other.  When working with `width %`, remember that only affects the width of the item relative to its container.  You set your `#content-wrapper` to width of `60%`? great.  seems legit.  BUT: it's children now (`#left` and `#right`) have widths of `50%` AND padding of `15%` for a total of `130%`.  Things spiral from there, as your `#text` div has a height of only 18%.  The contents spill over into the `#icons` div.  Remove the padding, remove the `#text` height, and go from there.